grc/core/cache.py

```diff
@@ -16,13 +16,14 @@
 
 class Cache(object):
 
-    def __init__(self, filename, version=None):
+    def __init__(self, filename, version=None, log=True):
```

Comment:
The changes in this file are made because the cache is used for the new example browser (see grc/gui_qt/components/example_browser.py).